packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -47,6 +47,9 @@ const SelectWrapper = styled(Box)`
   div[role='combobox'] {
     border: none;
     cursor: pointer;
+    min-height: unset;
+    padding-top: 6px;
+    padding-bottom: 6px;
```

Comment:
There was already custom modifications made on the components in the ToolBar for 2 main reasons (making it as small as possible so it doesn't waste to much space on the editing area and removing the borders around the IconButtons to avoid having multiple borders at the same place separated by small spacings). With the current Select, the Toolbar is bigger so it goes against the first point, and we end up with elements with multiple sizes in it. The cleanest thing would be of course to have more input sizes in the DS (I think we already discussed it), but in the meantime I'm sure it's easier to add another small override.

Reply:
Thanks for the reminder @remidej.

It's a shame we have to do this, but it shows another limitation of the DS imo. I agree, at least this is a consistent way of styling it, it'll be interesting to see if V2 is more flexible and i think this is a good case to feedback on when we get to it 👍🏼